phpunit.xml.dist

```diff
@@ -17,4 +17,8 @@
             <directory>src</directory>
         </whitelist>
     </filter>
+
+    <logging>
+        <log type="coverage-clover" target="build/coverage/clover.xml"/>
```

Comment:
forcing the collection of coverage data for anyone running the build looks wrong to me (and locally, I generally generate it in HTML format, not in clover format, as it is much easier to read for a human).
Thus, you are not even using this data on Travis, thus slowing down all builds for nothing.

I suggest you to use the command-line argument to generate coverage instead